docs/CMakeLists.txt

```diff
@@ -7,8 +11,40 @@ if (ENABLE_DOXYGEN_DOCS)
     configure_file(${DOXYGEN_IN} ${DOXYGEN_OUT} @ONLY)
 
     # note the option ALL which allows to build the docs together with the application
-    add_custom_target(doc_doxygen
+    add_custom_target(doxygen_docs
         COMMAND "${DOXYGEN_EXECUTABLE}" "${DOXYGEN_OUT}"
         WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
         COMMENT "Generating API documentation with Doxygen")
+    add_dependencies(docs doxygen_docs)
+    install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/html"
+        DESTINATION "${CMAKE_INSTALL_DOCDIR}"
+    )
+endif()
+
+if(ASCIIDOCTOR_EXECUTABLE)
+    add_custom_target(user_manual
+        COMMAND "${ASCIIDOCTOR_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/manual/manual.adoc" -o "${CMAKE_CURRENT_BINARY_DIR}/manual/manual.html"
+        COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_CURRENT_SOURCE_DIR}/manual/images" "${CMAKE_CURRENT_BINARY_DIR}/manual/images"
+        COMMENT "Generating OpenSCAP User Manual in HTML format"
+    )
+    install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/manual"
+        DESTINATION "${CMAKE_INSTALL_DOCDIR}"
+    )
+    add_custom_target(developer_manual
+        COMMAND "${ASCIIDOCTOR_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/developer/developer.adoc" -o "${CMAKE_CURRENT_BINARY_DIR}/developer/developer.html"
+        COMMENT "Generating OpenSCAP Developer Manual in HTML format"
+    )
+    install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/developer"
+        DESTINATION "${CMAKE_INSTALL_DOCDIR}"
```

Comment:
IMO it doesn't make sense to install the developer manual for end-users

Reply:
we should probably build it but not install it and add a comment explaining why we are not installing it

Reply:
aha ok